### PR TITLE
Refactor recommendations slightly

### PR DESCRIPTION
**Pull request by scwe**
Another conversion to use ES7 properly

### CODE REVIEW

File: src/utils/recommendations/get-recommendations.js [hunk 1]
```diff
@@ -6,19 +6,12 @@ import debug from 'debug'
...
     }
-    request(`${config.apiUri}/ct/rec/${appId}/VideoEnd`, options)
-      .then((recommendations) => {
-        log('recommendations', recommendations)
-        resolve(recommendations)
-      })
-      .catch(reject)
-  })
+  }
+  return await request(`${config.apiUri}/ct/rec/${appId}/VideoEnd`, options)
```
Comment (wojciechAdaszynski) [inline src/utils/recommendations/get-recommendations.js]: I know it's not related to this one but we should alway try...catch the parts where we would .then().catch() and we're not doing it in getRecommendations call

Comment (scwe) [inline src/utils/recommendations/get-recommendations.js]: I am confused as to how what I am doing is any different to what is happening previously. From what I understand ``.catch(reject)`` just throws an error anyway...

Comment (psmarcin) [inline src/utils/recommendations/get-recommendations.js]: again @scwe is right, but to be honest we should trigger our function for future integration with any tracking service

**Comment by psmarcin** [comment]
Same as #1

**Pull request merged**